capstone/src/components/User/User.jsx

```diff
@@ -0,0 +1,34 @@
+import './User.css';
+import { useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+import UserContext from '../../Contexts/UserContext';
+import Header from './Header/Header';
+
+export function userPage(userData) {
+  const navigate = useNavigate();
+  if (Object.keys(userData).length === 0) {
+    return (<h2>Loading ...</h2>);
+  }
+  if (userData.data.userType === 'adventurer') {
+    navigate('/adventurer');
```

Comment:
Will this change later? We discussed offline that this returns one JSX component while attempting to also update BrowserRouter.

Reply:
That one is weird, I checked that commit and I no longer have a User.jsx
I changed the navigate to the login, and afterwards I catched the path with the Route.
I can re-push the code with this whole feedback if that helps, so I can also get comments on the new part